[Security Solution] Adds posix eventing model fields to generator script

kqualters-elastic opened the pull request:
## Summary

This is the first in a series of prs to add the session view eventing model fields to Kibana for use by the new session view plugin. This pr is just to add the fields to the resolver data generator script. Fields will be present in each document, but will not have a mapping until the fields are actually merged to the ecs repo and published, and then the field mapping in rule registry updated. Any plugin that has a copy of the older Ecs type definition will need to update it's process field definition as well. Example of the events this change is intended to emulate can be found here https://github.com/elastic/ecs/pull/1779/files#diff-7127579f629c8343243eae0bc88fcad3cbfac7974d9d24e0fd44c3ee8c5d5d86R1903

Generated data in use in security solution:

![session_view_search](https://user-images.githubusercontent.com/56408403/157326961-988359c1-285c-4558-9db0-565d89223af7.gif)


### Checklist

- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

michaelolo24 reviewed (approved):
Thanks for adding this! Was able to confirm the fields show up as expected, but we're still waiting on them being added to the ecs mapping: 

<img width="1548" alt="image" src="https://user-images.githubusercontent.com/17211684/157503541-498d9b47-af3c-4155-8973-b77ecdb43494.png">

Also was able to verify the reason the names changed in the snapshot test were due to the pseudo-random number generator package seedrandom using a new value based on the added random string generation for the `sessionEntryLeader`.

Thanks for getting this done! LGTM!

kqualters-elastic commented:
@michaelolo24 for the fields to be mapped, they must be added here in ruleRegistry https://github.com/elastic/kibana/pull/127208/files#diff-a1647ccb73ef26c8c8b6aefd87084504b146af72fcb088ccacad93fcaad15b69R2404 . For the fields to be used in typescript, added here https://github.com/elastic/kibana/pull/127208/files#diff-7d6a5c41c8f067cdc16aa7d96811e68cb84f2042da7dabfc1117a430780922e3R31 and in any plugin using the non common Ecs type. For the fields to be returned by timeline search strategies, added here https://github.com/elastic/kibana/pull/127208/files#diff-b60a4cd35ce9623c3819a45f1ea51c4110ec15921e16d789e5693d73b18fbc12R216 . and finally for the field to always be returned so that removing a column does not exclude it from the result, add the field used by the action here https://github.com/elastic/kibana/pull/127208/files#diff-2676630f0684f0ba6f1d3e42850d059ded14bc72da00bf8e925bcc4ece71a87bR179
